openhtf/plugs/acroname/power_module_plug.py

```diff
@@ -0,0 +1,309 @@
+# Copyright 2014 Google Inc. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Acroname power module plug for OpenHTF."""
+
+
+import logging
+import time
+from time import sleep
+
+import openhtf.conf as conf
+import openhtf.plugs as plugs
+
+from brainstem import discover
+from brainstem.link import Spec
+from brainstem.stem import MTMPM1
+from brainstem.stem import MTMEtherStem
+from brainstem.stem import USBStem
+from brainstem.defs import model_info
+
+# _LOG = logging.getLogger("openhtf.example.plugs.acroname.power_module_plug.py")
+_LOG = logging.getLogger(__name__)
+
+conf.Declare('serial_number_of_EtherStem',
+             description='Serial number of EtherStem that connects PowerModule to Ethernet.')
+
+conf.Declare('default_power_module_voltage_output',
+             description='Defalut power module voltage output in uV.')
+
+conf.Declare('default_power_module_current_limit',
+             description='Defalut power module current limit in uA.')
+
+class NoneEtherStemFoundError(Exception):
```

Comment:
Some of these connection-related errors can probably be grouped together under a single type, like ```EtherStemConnectionError``` or similar. Then later when they're raised, instead of raising a super specific error type,  provide the detail through arguments. e.g.:

```
raise EtherSyncConnectionError('Failed to connect to power module through EtherStem.')
```